py/desispec/pixgroup.py

```diff
@@ -70,24 +70,30 @@ def get_exp2healpix_map(expfile, survey=None, program=None,
         specprod_dir = io.specprod_root()
 
     log.info(f'Reading exposures list from {expfile}')
-    if expfile.endswith( (".fits", ".fits.gz") ):
-        t = Table.read(expfile, 1)
-    else:
-        t = Table.read(expfile)
+    exps = Table.read(expfile, 'EXPOSURES')
+    frames = Table.read(expfile, 'FRAMES')
```

Comment:
Indeed.  To be clear, the reason for that is because the csv version of the file doesn't contain enough information (the CAMERA column of the FRAMES HDU) to be able to determine which petals exist.  This is another opportunity for (future) cleanup, to avoid a dependency on the exposures file (e.g. by standardizing the SURVEY and PROGRAM entries in the exposure_tables and then using those plus BADCAMWORD etc. parsing.